File: 8-kyu/simple-calculator-/python/solution.py
import operator

def calculator(x, y, op):
    if type(x) != int or type(y) != int or type(op) != str or op not in '+-*/':
        return 'unknown value'
    lookup = {'+': operator.add, '-': operator.sub,
             '*': operator.mul, '/': operator.truediv
    }
    return lookup[op](x, y)

File: 8-kyu/simple-calculator-/javascript/solution.js
const calculator = (a, b, sign) => {
  if (/[^\+\-\*\/]/g.test(sign) || isNaN(a) || isNaN(b)) {
    return "unknown value";
  }
  if (sign === "+") {
    return a + b;
  } else if (sign === "-") {
    return a - b;
  }
  else if (sign === "*") {
    return a * b;
  } else if (sign === "/" && b !== 0) {
    return a / b;
  }
};
